src/components/CardTable/CardTable.jsx

```diff
@@ -172,6 +178,9 @@ export default class CardTable extends Component {
     })
 
   handleWindowResize = () => new Promise((resolve) => {
+    if (!this.mounted) {
+      throw new Error('Not mounted')
+    }
```

Comment:
Let me know if your thoughts on using this. Borrowed the pattern from react-grid-layout [https://github.com/STRML/react-grid-layout/blob/f3e4a57766707e87886588cf2c0fb267ce33207c/lib/components/WidthProvider.jsx](https://github.com/STRML/react-grid-layout/blob/f3e4a57766707e87886588cf2c0fb267ce33207c/lib/components/WidthProvider.jsx)